<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/BackButtonIcon.png</normaloff>:/BackButtonIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="MuteButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/MuteButtonIcon.jpg</normaloff>:/MuteButtonIcon.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>85</width>
     <height>110</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ToggleLEDButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/ToggleLEDButtonIcon.PNG</normaloff>:/ToggleLEDButtonIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>161</width>
     <height>161</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ToggleWaterButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/ToggleWaterButtonIcon.PNG</normaloff>:/ToggleWaterButtonIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ManualWaterButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>171</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/ManualWaterIcon.PNG</normaloff>:/ManualWaterIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="WaterIcon">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>32</y>
     <width>101</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources/biobloom.qrc">:/WaterDroplet.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="RedLEDButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/RedLEDIcon.PNG</normaloff>:/RedLEDIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>70</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="WhiteLEDButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/WhiteLEDIcon.PNG</normaloff>:/WhiteLEDIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>70</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BlueLEDButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>230</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/BlueLEDIcon.PNG</normaloff>:/BlueLEDIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>70</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CyanLEDButton">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>310</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/CyanLEDIcon.PNG</normaloff>:/CyanLEDIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>70</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="YellowLEDButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>310</y>
     <width>90</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/YellowLEDIcon.PNG</normaloff>:/YellowLEDIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>80</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="GreenLEDButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>410</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/GreenLEDIcon.PNG</normaloff>:/GreenLEDIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>70</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="MagentaLEDButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources/biobloom.qrc">
     <normaloff>:/MagentaLEDIcon.PNG</normaloff>:/MagentaLEDIcon.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>80</width>
     <height>70</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Resources/biobloom.qrc"/>
 </resources>
 <connections/>
</ui>
